src/app/modules/gb-navbar-module/gb-navbar.component.css

```diff
@@ -7,7 +7,7 @@
 }
 
 /* Style the links inside the navigation bar */
-.gb-nav a {
+.gb-nav a, .gb-nav-element {
```

Comment:
not sure what this does, originally this is to style the `a` element in navbar, not the entire navbar element

Reply:
This is to also style other elements in the navbar than anchors. In this case, a warning icon.